apis/configuration/slo_controller_config.go

```diff
@@ -295,6 +296,31 @@ type ModelRatioCfg struct {
 	HyperThreadTurboEnabledRatio *float64 `json:"hyperThreadTurboEnabledRatio,omitempty"`
 }
 
+// AmplificationCfg is the cluster-level configuration of the resource amplification strategy.
+// +k8s:deepcopy-gen=true
+type AmplificationCfg struct {
+	AmplificationStrategy `json:",inline"`
+	NodeConfigs           []NodeAmplificationCfg `json:"nodeConfigs,omitempty" validate:"dive"`
+}
+
+// NodeAmplificationCfg is the node-level configuration of the resource amplification strategy.
+// +k8s:deepcopy-gen=true
+type NodeAmplificationCfg struct {
+	NodeCfgProfile `json:",inline"`
+	AmplificationStrategy
+}
+
+// AmplificationStrategy is the resource amplification strategy.
+// +k8s:deepcopy-gen=true
+type AmplificationStrategy struct {
+	// Enable defines whether the resource amplification strategy is enabled.
+	Enable *bool `json:"enable,omitempty"`
+	// CPURatio defines the cpu amplification ratio.
+	CPURatio *float64 `json:"CPURatio,omitempty"`
```

Comment:
How about use a more general `ResourceRatioMap map[corev1.ResourceName]float64`?

Reply:
> How about use a more general `ResourceRatioMap map[corev1.ResourceName]float64`?

Good Idea, and all is done.